src/Microsoft.IO.Redist/src/Resources/Strings.resx

```diff
@@ -0,0 +1,291 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Any reason you decided to copy rather than link this file?

Reply:
Yeah, it has resources from Core as well. I could add the few strings to CoreFX (S.IO.FS), but they would take up space.

Reply:
Oh, I see, some may be from corelib some from IO.  Bummer that we have multiple instances of this stuff, but I guess this isn't the first case.  We probably could do better with this stuff.  Can you at least clean up the file of the boilerplate comments and do your best to ensure its a minimal set of strings?

Reply:
Sure. I'll trim it down.